examples/actors/array_ro_csv.yaml

```diff
@@ -0,0 +1,25 @@
+---
+inputs:
+  - name: string_list
+    type: TypeStringList
+  
+outputs:
+  - name: csv_string 
+    type: BaseTypeString
+
+executor:
+  type: python
+  payload: |
+    from json import load, dumps
+    import sys
+
+    inputs = load(sys.stdin).get("string_list").get("value", [])
```

Comment:
if you use get to avoid the exception, then you have to use .get("string_list", {}).get("value", []) otherwise it will raise an AttributeError that None has no attribute called get

Reply:
load(sys.stdin)["string_list"]["value"] is sufficient, there is a requirement that these fields exist in the input - If this fails - this is fine that there's a KeyError that will cause the processing to fail